decidim-proposals/app/views/decidim/proposals/admin/proposals/_form.html.erb

```diff
@@ -26,7 +26,10 @@
 
     <% if current_participatory_space.has_subscopes? %>
       <div class="row column">
-        <%= form.scopes_select :scope_id, prompt: I18n.t("decidim.scopes.global"), remote_path: decidim.scopes_search_path(root: current_participatory_space.scope) %>
+        <%= form.scopes_picker :scope_id do |scope|
+          { url: decidim.scopes_picker_path(root: current_participatory_space.scope, current: scope&.id, field: form.label_for(:scope_id)),
```

Comment:
I see this repeated in other files, what do you think about moving this logic to a helper method?

Reply:
It definitely looks like it should go to a helper in core!

Reply:
Yes, you are totally right, I have had to make multiple replacements many times and I didn't see it :disappointed:.